### PR TITLE
Added information about images in html

### PR DESCRIPTION
**Pull request by prankshaw**
Explained  tag completely with example an article link for further updates

### CODE REVIEW

File: src/pages/html/tutorials/images-in-html/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Images in HTML
 ---
+
 ## Images in HTML
 
 This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/tutorials/images-in-html/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```
Comment (amandeepmittal) [inline src/pages/html/tutorials/images-in-html/index.md]: Also, please remove this line "This is a stub..."

**Comment by amandeepmittal** [comment]
Never mind @prankshaw. I have resolved the conflicts.

**Comment by prankshaw** [comment]
I am sorry @amandeepmittal . I will make sure this won't happen again. And thanks for merging

**Comment by luandy64** [comment]
Hey @amandeepmittal,

In the conflicts, you didn't address this tag that isn't rendering. 

Do you think we should remove the `` and the `-->` or remove the `-->` and add backticks around image tag?

screen shot 2018-07-21 at 2 44 03 pm

**Comment by prankshaw** [comment]
Allow me to do this @amandeepmittal  @luandy64 . I will remove -> and img tag to be visible. If there are any more changes, please inform
Just as I will get confirmation, I will make changes

**Pull request merged**